src/doc/src/reference/build-scripts.md

```diff
@@ -258,12 +258,16 @@ identifier, the value should be a string.
 
 ### `cargo::rustc-check-cfg=CHECK_CFG` {#rustc-check-cfg}
 
+*See the announcement [blog post][check-cfg-blog-post] for more global view of the feature.*
+
 Add to the list of expected config names and values that is used when checking
 the _reachable_ cfg expressions.
 
 For details on the syntax of `CHECK_CFG`, see `rustc` [`--check-cfg` flag][option-check-cfg].
 See also the [`unexpected_cfgs`][unexpected-cfgs] lint.
 
+> Note: `cargo:rustc-check-cfg` can be used if your MSRV is below Rust 1.77
```

Comment:
```suggestion
> Note: `cargo:rustc-check-cfg` (single-colon) can be used if your MSRV is below Rust 1.77
```

Also, while this trick works for most of the allowed invocations, people would likely hit `--check-cfg` lint error and got here. Hence makes sense addding one here.